se_feedback/sprint-5/feedback_sprint_5_all.md

```diff
@@ -0,0 +1,97 @@
+|      |            |
+|------|------------|
+|Group | Rubber Duck Debuggers |
+|Assignment|Sprint 5|
+|Date|29/05/16|
+|TA|Bastiaan Reijm|
+
+#Sprint Feedback
+Feedback and Grades for Sprint 5.
+
+Total: **9.55**
+
+| User Story | Score |
+|------------|-------|
+| definition |  9   |
+| splitting  |  10     |
+| responsibility | 10  |
+
+| Learning from History | Score |
+|-----------------------|-------|
+| estimation            | 10    |
+| prioritisation        | 10    |
+| reflection            | 9   |
+
+## Notes
+* Well done
+* That Matthias took over Robin's task should be added in a note, it looks like both were assigned responsibility and then one member was removed
```

Comment:
Dear Mathias,

Thanks for pointing out the extra non-existent group member. I'll be sure to keep him out of future sprints for the reasons you mentioned.

Regards,
Bastiaan